crates/sui-indexer/src/store/pg_indexer_store.rs

```diff
@@ -178,18 +183,22 @@ impl<T: R2D2Connection + 'static> PgIndexerStore<T> {
         transactional_blocking_with_retry!(
             &self.blocking_cp,
             |conn| {
-                diesel::insert_into(display::table)
-                    .values(display_updates.values().collect::<Vec<_>>())
-                    .on_conflict(display::object_type)
-                    .do_update()
-                    .set((
+                on_conflict_do_update!(
```

Comment:
nit: no strong opinion, but I feel it's slightly more readable to have separate macros for pg and mysql, although it might cause a bit more duplicate codes.